modules/ROOT/pages/_partials/endpoint-sftp-receive.adoc

```diff
@@ -0,0 +1,69 @@
+== SFTP Receive Settings
+
+
+=== Server information
+
+[%header,cols="3s,7a"]
+|===
+|Name |Description
+
+|Host
+| Enter the Fully Qualified Domain Name (FQDN) or IP address of the server on your host machine that connects to the external SFTP server. Ability to provide the “Host”. Enable free text input. +
```

Comment:
no caps for fully qualified domain name?

Reply:
Why is Host in quotes? "Enable free text input" is without context, no subject, hard to understand what it means.

Reply:
Need further investigation here.